core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryMetrics.java

```diff
@@ -35,9 +35,11 @@ public class JsonQueryMetrics implements Mutable {
     private final Counter cacheMissCounter;
     private final LongGauge cachedQueriesGauge;
     private final Counter completedQueriesCounter;
+    private final LongGauge connectionsGauge;
     private final Counter startedQueriesCounter;
 
     public JsonQueryMetrics(MetricsRegistry metricsRegistry) {
+        this.connectionsGauge = metricsRegistry.newLongGauge("json_queries_connections");
```

Comment:
nit: could we rename this to "connectionCountGauge", "connections" is not very descriptive

Reply:
done in 705474eb227d0c7b10b9b6fa8a39b7d0f6bb1830